src/v1/item/utils/isOpen.ts

```diff
@@ -0,0 +1,4 @@
+import { Item, Mapped } from 'v1/item'
+
+export const isOpen = ({ _open }: Item | Mapped): boolean => _open
```

Comment:
wdym by open and closed here?

mutable and immutable? 🤔

Reply:
It means "accepts additional properties".

So for instance, `map({ foo: string() })` rejects a `bar` property but `map({ foo: string() }).open()` accepts it.

We can search for a better wording if you want !

Reply:
Maybe strict?